### PR TITLE
added link to spoof intelligence insight

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#1167

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/create-safe-sender-lists-in-office-365.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ The recommended method to configure a safe sender list is to use mail flow rules
 > [!IMPORTANT]
 > Information on how to create a **Blocked Sender List** is [here](create-block-sender-lists-in-office-365.md).
 
+> [!NOTE]
+> To allow sender domain to send unauthenticated emails (bypass anti-spoofing protection), but not bypassing anti-spam and anti-malware checks, you can add it to [AllowedToSpoof safe sender list]( https://docs.microsoft.com/microsoft-365/security/office-365-security/walkthrough-spoof-intelligence-insight).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/create-safe-sender-lists-in-office-365.md]:
```suggestion
> To allow a sender domain to send unauthenticated emails (bypass anti-spoofing protection) but not bypass anti-spam and anti-malware checks, you can add it to the [AllowedToSpoof safe sender list]( https://docs.microsoft.com/microsoft-365/security/office-365-security/walkthrough-spoof-intelligence-insight).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #1167.

Thank you.

cc: @kenwith

**Pull request merged**